networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/ActiveEth2P2PNetwork.java

```diff
@@ -142,12 +144,20 @@ private synchronized void startGossip() {
             discoveryNetwork::setSyncCommitteeSubnetSubscriptions);
 
     gossipForkManager.configureGossipForEpoch(recentChainData.getCurrentEpoch().orElseThrow());
-    processedAttestationSubscriptionProvider.subscribe(gossipForkManager::publishAttestation);
-    eventChannels.subscribe(BlockGossipChannel.class, gossipForkManager::publishBlock);
 
     setTopicScoringParams();
   }
 
+  private synchronized void stopGossip() {
+    if (gossipStarted.compareAndSet(true, false)) {
+      LOG.warn("Stopping eth2 gossip while node is syncing");
```

Comment:
warn on the basis we can't perform operations?

Reply:
Warn because we always have. :) Mostly it's just useful to know the node is no longer following gossip. There's probably an argument it could be info but it is an indication that something isn't ideal (the node is significantly out of sync when it was previously in sync).